source/features/fix-view-file-link-in-pr.tsx

```diff
@@ -24,13 +23,19 @@ function handleMenuOpening(event: delegate.Event): void {
 		return;
 	}
 
+	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
+
+	// If the file was deleted
+	if (viewFile.closest('[data-file-deleted="true"]')) {
+		return;
+	}
+
 	// This solution accounts for:
 	// - Branches with slashes in it
 	// - PRs opened from the default branch
-	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
-	const pathParts = viewFile.pathname.split('/'); // Example pathname: $owner/$repository/blob/$sha/$path_to_file.tsx
-	pathParts[4] = getCurrentBranch();
-	viewFile.pathname = pathParts.join('/');
+	const headRepo = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname;
+	const filepath = viewFile.pathname.split('/').slice(5).join('/'); // Example pathname: $owner/$repository/blob/$sha/$path_to_file.tsx
```

Comment:
```suggestion
	const filepath = dropdown.closest('[data-path]').dataset.path;
```

It appears to work even after renames. Example URL: https://github.com/sindresorhus/refined-github/pull/1750/files#diff-15c8cc76e8bb627ef332f633511c9c38